Write documentation for `daml test` coverage report (#16024)
digital-asset/daml: docs/source/daml/intro/11_StdLib.rst

Next Up
-------

> In the following section, you'll learn a little more about your options for testing and interacting with Daml code in :doc:`12_Testing`, about the operational semantics of some keywords and common associated failures, and a little bit about how coverage reports work in Daml testing.

Reviewer: Note: We are restructuring menus and I'm not sure if this navigation signpost will be correct. We are going to have to do a big overhaul of all this sort of content, fyi.

```suggestion
In the following section, you'll find options for testing and interacting with Daml code`. We also talk about the operational semantics of some keywords and their commonly associated failures, and a little bit about how coverage reports work in Daml testing.
```
